airbyte-integrations/connectors/source-chartmogul/acceptance-test-config.yml

```diff
@@ -4,26 +4,27 @@ connector_image: airbyte/source-chartmogul:dev
 acceptance_tests:
   spec:
     tests:
-      - spec_path: "source_chartmogul/spec.yaml"
+    - spec_path: "source_chartmogul/spec.yaml"
   connection:
     tests:
-      - config_path: "secrets/config.json"
-        status: "succeed"
-      - config_path: "integration_tests/invalid_config.json"
-        status: "failed"
+    - config_path: "secrets/config.json"
+      status: "succeed"
+    - config_path: "integration_tests/invalid_config.json"
+      status: "failed"
   discovery:
     tests:
-      - config_path: "secrets/config.json"
+    - config_path: "secrets/config.json"
   basic_read:
     tests:
-      - config_path: "secrets/config.json"
-        configured_catalog_path: "integration_tests/configured_catalog.json"
-        expect_records:
-          path: "integration_tests/expected_records.jsonl"
-          extra_fields: no
-          exact_order: no
-          extra_records: yes
+    - config_path: "secrets/config.json"
+      configured_catalog_path: "integration_tests/configured_catalog.json"
+      expect_records:
+        path: "integration_tests/expected_records.jsonl"
+        extra_fields: no
+        exact_order: no
+        extra_records: yes
+      fail_on_extra_columns: false
```

Comment:
I spot-checked these, but I have low confidence that I didn't miss a tab or space 💀